.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "plugins": ["redos-detector"],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": { "ecmaVersion": "latest" },
+  "rules": {
+    "redos-detector/no-unsafe-regex": ["error"],
```

Comment:
What are the advantages of this plugin compared, for example, with [`eslint-plugin-redos`](https://makenowjust-labs.github.io/recheck/docs/usage/as-eslint-plugin/)?

Reply:
They take different approaches. With mine the goal was to never report a pattern as safe when it's not, so it may have false positives

Not sure if they have the same goal. With a fuzzing approach I think there is the potential to miss things but looks like sometimes it also uses an automation which maybe can't miss stuff

Reply:
for what it's worth, looks like that one doesn't handle back references as well

```js
/(a)\1*$/
```

is a false positive with that one

Reply:
@tjenkinson I'm the author of `eslint-plugin-redos`, and, I found this comment luckily ;)

`/(a)\1*$/` seems a vulnerable regex to me and my detector also reports it as vulnerable, so what is it problem?
(If you feel that discussion here is not beneficial, please email to me or open a new issue to [makenowjust-labs/recheck](https://github.com/makenowjust-labs/recheck) instead.)

Reply:
Hey @makenowjust !

`/(a)\1*$/` is essentially `/aa*$/` which is not vulnerable?

Reply:
Ah I see maybe it is without the start anchor 🤔

Reply:
Yes. `/(a)\1*$/` is essentially `/^.*aa*$/s`, so it is vulnerable.